ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/CanHardFork.hs

```diff
@@ -563,11 +654,12 @@ translateChainDepStateShelleyToAllegraWrapper ::
        (ShelleyBlock (AllegraEra c))
 translateChainDepStateShelleyToAllegraWrapper =
     ignoringBoth $
-      Translate $ \_epochNo (WrapChainDepState stateShelley) ->
-        WrapChainDepState stateShelley
+      Translate $ \_epochNo (WrapChainDepState chainDepState) ->
+        -- Same protocol, same 'ChainDepState'
```

Comment:
Comment: we don't have to apply any epoch-related changes, this happens when ticking, as usual.

Reply:
I also realised that we don't need specific translations for `ChainDepState` and `LedgerView`.